bin/process.sh

```diff
@@ -6,10 +6,15 @@ shift $[$#-2]
 
 input=$1
 output=$2
-# target size in KB
-THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300}
+
+THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300} # target size in KB
 TILER_BASE_URL=${TILER_BASE_URL:-http://tiles.openaerialmap.org}
 
+# Although TMPDIR is usually set by the OS, it doesn't seem to be getting passed through
+# Docker and Node's `spawn()`, so no harm done in just defaulting to `/tmp` in such cases.
+# There may be clues here as the underlying issue: https://github.com/npm/npm/issues/4531
+TMPDIR={TMPDIR:-/tmp}
```

Comment:
Absolutely. Of course I'd need to see the specific `gdal` requirements, but I'd be surprised if most of the transcoder's features won't work with a default version. I'm thinking about the ability for devs to get up and running quickly and for the ease of writing tests. Besides, I think it's fine to have a Dockerfile for a specific use case, just not as a non-negotiable dependency.